userenvs/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "You must provide a directory"
+    exit
+fi
+
+userenv="$1"
+if [ ! -d "$userenv" ]; then
+    echo "Directory $userenv not found"
+    exit 1
+fi
+
+
+pushd "$userenv"
+echo "{" >../staging.json
```

Comment:
I'm just curious, does `jq` allow you to manipulate/create JSON docs easier than manually doing all these `echo` commands?

Reply:
I think it is strictly query, but there may be a program for that.